reconcile/utils/terrascript_aws_client.py

```diff
@@ -2461,6 +2462,25 @@ def populate_tf_resource_role(self, spec):
         role_tf_resource = aws_iam_role(identifier, **values)
         tf_resources.append(role_tf_resource)
 
+        role_policy = common_values.get("role_policy")
+        if role_policy:
```

Comment:
since `role_policy` only used in `if` block, can be

```python
if role_policy := common_values.get("role_policy")
```

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4038/commits/ebd6a58459e85fe19289b8bdffdedb7e742b2545